README.md

```diff
@@ -684,22 +685,45 @@ Special characters can be included:
 * Single Quote: `\'`
 * Unicode scalar: `\u{n}` where n is between one and eight hexadecimal digits
 
-#### Array Access Syntax
+The built-in methods and accessors for Strings are different in Swift 1.x, 2.1+, and 3.0, and so are not covered here.
+
+#### Array access syntax
 
 ```swift
+// single item access
 let example = [ "hi", "there", 23, true ]
 print("item at index 0: \(example[0])")
+
+// range access
+var fibonacci = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 5]
+fibonacci[5..8] // [5, 8, 13]. Note: the end range value is excluded
+fibonacci[0..fibonacci.endIndex] // all except last item
```

Comment:
The above two lines don't compile; please ensure you check all code examples in an Xcode 8+ Playground before submitting